src/main/app/claims/models/claim.ts

```diff
@@ -111,4 +113,22 @@ export class Claim {
 
     return !this.countyCourtJudgmentRequestedAt && this.remainingDays < 0 && !this.respondedAt
   }
+
+  get status (): ClaimStatus {
+    if (this.countyCourtJudgmentRequestedAt) {
+      return ClaimStatus.CCJ_REQUESTED
+    } else if (this.eligibleForCCJ) {
+      return ClaimStatus.ELIGIBLE_FOR_CCJ
+    } else if (toBoolean(config.get<boolean>('featureToggles.offer')) && this.settlement && this.response.responseType === ResponseType.FULL_DEFENCE) {
```

Comment:
You can use `FeatureToggles.isEnabled('offer')` instead. See `featureToggle.ts`.